data/broker-features.js

```diff
@@ -0,0 +1,255 @@
+export default [
+    {
+        feature: "Produce / Exclusive",
+        stableSince: "2.2.0",
```

Comment:
The exclusive producer was released in 2.11.0
https://github.com/apache/pulsar/issues/15528
I think we can mark it as stable since 3.0.0

Reply:
Thank you!